python/mlx/nn/layers/quantized.py

```diff
@@ -268,3 +268,93 @@ def from_linear(
             ql.bias = linear_layer.bias
 
         return ql
+
+
+class QQLinear(Module):
```

Comment:
Does it make sense to hijack `Module::train()` and `Module::eval()` to update the weights between quantize and not-quantize?

I think at the very least it would be nice to have a method which turns it into the inference version since I expect it would be a common use case.

CC @angeloskath might have some ideas.